### ISSUE TITLE
Option in admin when adding to cart to open or not the cart drawer on add to cart

### ISSUE THREAD

**Issue by vchiriacmoldoweb** [monsoonconsulting/magento2-hyva-ajax-add-to-cart#21]
**Is your feature request related to a problem? Please describe.**

- This module works pretty nice, it would be nice to offer this feature because it would complete it in my perspective, to be honest I need to for my client and I can implement this on my own but since its a feature module I think it would be best to be included in.

**Describe the solution you'd like**

- Add a system configuration with "Open cart on add to cart trigger" or something similar.
- Add the check in the script present in the page and if yes then try to integrate one of the 2 solutions:
     - trigger a click on the minicart
     - try to work with the "$dispatch('toggle-cart', { isOpen: true })" 

**Describe alternatives you've considered**

**Additional context**

**Comment by callanzimmermann** [comment]
Hello @vchiriacmoldoweb, I've made some changes to implement this, can you pull the dev branch and check the changes from your side before I merge it?
```bash
composer require monsoonconsulting/magento2-hyva-ajax-add-to-cart:dev-dev
```

**Comment by vchiriacmoldoweb** [comment]
@callanzimmermann works pefectly!!!! Thank you very much for this implementation, now its complete :D If I find any issues I will let you know.

**Comment by callanzimmermann** [comment]
Hello @vchiriacmoldoweb, that's great. I will do some further testing and merge it to main hopefully in the next week.

**Comment by callanzimmermann** [comment]
Hello @vchiriacmoldoweb, available in 1.2.1

**Comment by vchiriacmoldoweb** [comment]
@callanzimmermann please check https://github.com/monsoonconsulting/magento2-hyva-ajax-add-to-cart/issues/25, a new bug was introduced
